feat(RDS): add a resource to upgrade instance minor version (#6935)
huaweicloud/terraform-provider-huaweicloud: docs/resources/rds_upgrading_minor_version.md

---
subcategory: "Relational Database Service (RDS)"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_rds_upgrading_minor_version"
description: |-

>   Use this resource to perform a kernel minor version upgrade on a MySQL or PostgreSQL RDS instance.

Reviewer: ```suggestion
  Managers an RDS instance minor version upgrade resource within HuaweiCloud.
```

---

# huaweicloud_rds_upgrading_minor_version

Use this resource to perform a kernel minor version upgrade on a MySQL or PostgreSQL RDS instance.

## Example Usage

```hcl
variable "instance_id" {}

resource "huaweicloud_rds_upgrading_minor_version" "test" {
  instance_id = var.instance_id
}
```

## Argument Reference

The following arguments are supported:

* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
  If omitted, the provider-level region will be used.

> * `instance_id` - (Required, String, ForceNew) Specifies the instance ID.

Reviewer: ```suggestion
* `instance_id` - (Required, String, NonUpdatable) Specifies the instance ID.
```

> * `is_delayed` - (Optional, Boolean) Specifies whether the upgrade is delayed to the maintenance window.

Reviewer: ```suggestion
* `is_delayed` - (Optional, Bool, NonUpdatable) Specifies whether the upgrade is delayed to the maintenance window.
```

  + **true**: Specifies the upgrade is delayed and performed within the maintenance window.
  + **false**: Specifies the upgrade is performed immediately. This is the default value.

## Attribute Reference

In addition to all arguments above, the following attributes are exported:

* `id` - The resource ID.

> * `job_id` - Indicates the task ID for the upgrade operation, which can be used to track the job status.

Reviewer: should be deleted
